### ISSUE TITLE
Java language level 

### ISSUE THREAD

**Issue by PiekJ** [yarwest/Yet-Another-Coding-Guideline#2]
The Java language level should be at set at 8, and the java version at 1.8.

**Comment by yarwest** [comment]
I do not think that this is something that should be taken into this guideline since it is project dependent and it might be something that becomes outdated really quickly while the guide it pushing to be a lasting resource.

It is basically like saying: "You need to use Angular 2".
